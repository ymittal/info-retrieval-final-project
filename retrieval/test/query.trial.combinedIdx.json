{
    "casefold": true,
    "fields": [
        "text",
        "title",
        "headline",
        "head",
        "leadpara"
    ],
    "verbose": true,
    "index": ["../../idx/tedDirector_en.idx", "../../idx/tedX_en-ALL.idx", "../../idx/tedEd_en-ALL.idx"],
    "requested": 5,
    "queries": [{
        "number": "inTEDx",
        "text":"solvesthem",
        "comment":"extracted from: d0DrguC0Flc"
      },{
        "number":"inTedEd",
        "text":"osteoblasts",
        "comment":"extracted from: yJoQj5-TIvE"
      }]
}
