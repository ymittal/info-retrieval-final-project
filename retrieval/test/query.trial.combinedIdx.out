inTEDed Q0 rO6S93FKIUM 1 -6.78382180 galago
inTEDed2 Q0 B5x5CnwQZD4 1 -5.70855270 galago
inTEDed2 Q0 tVdw60eCnJI 2 -5.76166252 galago
